source/OpenBveApi/Objects/ObjectTypes/AnimatedWorldObject.cs

```diff
@@ -1,20 +1,25 @@
-﻿using OpenBveApi.Trains;
+﻿using System;
+using OpenBveApi.Hosts;
+using OpenBveApi.Trains;
 
 namespace OpenBveApi.Objects
 {
 	/// <summary>A container object for animated objects within the world</summary>
 	public class AnimatedWorldObject : WorldObject
 	{
-		/// <summary>Holds a reference to the host application</summary>
-		private readonly Hosts.HostInterface currentHost;
 		/// <summary>The signalling section the object refers to (Only relevant for objects placed using Track.Sig</summary>
 		public int SectionIndex;
 
 		/// <summary>Creates a new Animated World Object</summary>
 		/// <param name="Host">The host application</param>
-		public AnimatedWorldObject(Hosts.HostInterface Host)
+		public AnimatedWorldObject(HostInterface Host) : base(Host)
```

Comment:
The Hosts.HostInterface was actually deliberate and is likely to be required. (Mono compilation sometimes doesn't seem to like it otherwise)

See what CI does.

Reply:
I did not know that.
As far as I can see from the CI results, the build seems to be successful, but I will send this part back.

Reply:
If it builds. don't worry about it I'd say (Although it's repeated in a bunch of places)

I think it was the original makefile / MCS from memory.

Reply:
OK.
I fixed it.

Reply:
Oops, I found a missing fix.

Reply:
I think it has been fixed.